submissions/aymkin/js-dom/index.js

```diff
@@ -0,0 +1,126 @@
+  },
+  {
+    name: "Banana",
+    imgSrc: "./img/banana.png",
+    description:
+      'A banana is an edible fruit – botanically a berry[1][2] – produced by several kinds of large herbaceous flowering plants in the genus Musa.[3] In some countries, bananas used for cooking may be called "plantains", distinguishing them from dessert bananas. The fruit is variable in size, color, and firmness, but is usually elongated and curved, with soft flesh rich in starch covered with a rind, which may be green, yellow, red, purple, or brown when ripe.'
+  },
+  {
+    name: "Grape",
+    imgSrc: "./img/grape.png",
+    description:
+      "A grape is a fruit, botanically a berry, of the deciduous woody vines of the flowering plant genus Vitis.\n\nGrapes can be eaten fresh as table grapes or they can be used for making wine, jam, juice, jelly, grape seed extract, raisins, vinegar, and grape seed oil. Grapes are a non-climacteric type of fruit, generally occurring in clusters."
+  },
+  {
+    name: "Pear",
+    imgSrc: "./img/pear.png",
+    description:
+      "The pear (/ˈpɛər/) tree and shrub are a species of genus Pyrus /ˈpaɪrəs/, in the family Rosaceae, bearing the pomaceous fruit of the same name. Several species of pear are valued for their edible fruit and juices while others are cultivated as trees."
+  },
+  {
+    name: "Durian",
+    imgSrc: "./img/durian.png",
+    description:
+      "The durian (/ˈdjʊəriən, ˈdʊr-, -æn/)[2] is the fruit of several tree species belonging to the genus Durio. There are 30 recognised Durio species, at least nine of which produce edible fruit, with over 100 named varieties in Indonesia,[3] 300 in Thailand and 100 in Malaysia.[4] Durio zibethinus is the only species available in the international market: other species are sold in their local regions."
+  }
+];
+
+const doc = document;
+
+const container = doc.createElement("div");
+container.setAttribute("class", "container");
+
+const pageName = doc.createElement("h1");
+pageName.setAttribute("id", "page-name");
+pageName.textContent = "JS DOM";
+
+const dictionaryName = doc.createElement("h2");
+dictionaryName.setAttribute("id", "dictionary-name");
+dictionaryName.textContent = "Fruit dictionary";
+
+const divMain = doc.createElement("div");
+divMain.setAttribute("class", "main");
+
+const divMenu = doc.createElement("div");
+divMenu.setAttribute("class", "menu");
+
+const navigation = doc.createElement("ul");
+navigation.setAttribute("id", "navigation");
+const nav = doc.querySelectorAll("li");
+
+for (let i = 1; i <= fruits.length; i += 1) {
+  const li = doc.createElement("li");
+  li.setAttribute("class", "list-element");
+  li.textContent = fruits[i - 1].name;
+  if (i - 1 === 0) {
```

Comment:
It doesn't look very nice `fruits[i - 1]`
Start iterating from scratch!

Reply:
Done

Reply:
Values ​​in arrays start from zero indexes; therefore, it is better to assign zero and not one to the element, thus not bothering with the operation of taking away at each iteration.
```javascript
for(let i=0; i<fruits.length; i++) {
      fruits[i];
      .......
}
```
And best of all, try to iterate over an array with a function `forEach().`
```javascript
fruits.forEach(function(val, key) {
       console.log(key + " <--> " + val);
       if (key === 0) { ... }
})
```

Reply:
Done, now is used foreach instead of for